docs/UserGuide.md

```diff
@@ -252,12 +252,15 @@ To delete the module:
 1. Type `delete cs2107` into the command box, and press **Enter** to execute it.<br>
 ![Delete1](images/Delete1.png)
 
-2. The result display will display the message "Deleted Module" along with the details of the module you deleted.
+2. When you are prompted for a confirmation, type `yes` into the command box, and press **Enter**.<br>
 ![Delete2](images/Delete2.png)
 
-3. You can then confirm that the module no longer exists in the Completed Modules list by `find`-ing the module Type `find cs2107` into the command box, and press **Enter** to execute it. You will see that you can no longer `find` the module in the Completed Modules list.<br>
+3. The result display will display the message "The following module has been successfully deleted" along with the details of the module you deleted.
 ![Delete3](images/Delete3.png)
+
+4. You can then confirm that the module no longer exists in the Completed Modules list by `find`-ing the module. Type `find cs2107` into the command box, and press **Enter** to execute it. You will see that you can no longer `find` the module in the Completed Modules list.<br>
```

Comment:
Perhaps we can leave this out? Not really relevant to the delete command

Reply:
Agreed - I think this could be in the testing guide in our DG instead since it's like "confirming" that the delete works. But I'm not too concerned abt this for the PE-dry-run for now :D

Reply:
done!